vars/fcosKolaTestIso.groovy

```diff
@@ -0,0 +1,34 @@
+// Run kola testiso
+// Available parameters:
+//     cosaDir:        string  -- cosa working directory
+//     extraArgs:      string  -- extra arguments to pass to `kola testiso`
+//     extraArgs4k:    string  -- extra arguments to pass to 4k `kola testiso`
+//     skipMetal4k:    boolean -- skip metal4k image
+def call(params = [:]) {
+    def cosaDir = utils.getCosaDir(params)
+    def extraArgs = params.get('extraArgs', "");
+    def extraArgs4k = params.get('extraArgs4k', "");
+
+    testIsoRuns = [:]
+    testIsoRuns["Metal"] = {
+        try {
+            shwrap("cd ${cosaDir}  && kola testiso -S ${extraArgs} --output-dir tmp/kola-testiso-metal")
+        } finally {
+            shwrap("cd ${cosaDir} && tar -cf - tmp/kola-testiso-metal/ | xz -c9 > ${env.WORKSPACE}/kola-testiso-metal.tar.xz")
+        }
+    }
+    if (!params['skipMetal4k']) {
+        testIsoRuns["Metal4k"] = {
+            try {
+                shwrap("cd ${cosaDir} &&  kola testiso -S --qemu-native-4k ${extraArgs4k} --output-dir tmp/kola-testiso-metal4k")
+            } finally {
+                shwrap("cd ${cosaDir} && tar -cf - tmp/kola-testiso-metal4k/ | xz -c9 > ${env.WORKSPACE}/kola-testiso-metal4k.tar.xz")
```

Comment:
I think we need `--no-pxe` here since `kola testiso` doesn't support testing metal4k + PXE yet.

Reply:
I'm thinking to pass it as argument.
In RHCOs for example we have:
`kola testiso -S --qemu-native-4k --scenarios iso-install --output-dir tmp/kola-metal4k`
It will be more flexible, what do you think?

Reply:
Sure SGTM. Was thinking we could have a `scenarios` variable to make this nicer, but I think it's good anyway to have an `extraArgs` escape hatch. So we can add a `scenarios` var as well as a follow-up. (Or if you feel like it, feel free!)